index.html

```diff
@@ -54,7 +54,170 @@ <h2 id="name_title">I’m Chima Glad to see you!</h2>
                     </ul>
                 </nav>
             </div>                        
-        </section>        
+        </section>      
+        <section id="works">
+            <div class="card">
+                <a href="#" class="project_cta_link">
+                    <img src="assets/images/SnapshootPortfolio.png" alt="Project Name" class="snapshot_portfolio">
+                </a>
+                <div class="project_description">
+                    <div class="project_title">Tonic</div>
+                    <div class="project_summary">
```

Comment:
- [ ] The project title (e.g., "Tonic") uses a header tag hx, seeing yours, you are wrapping it inside a div element, please change it.